doc/neps/nep-0050-scalar-promotion.rst

```diff
@@ -77,9 +77,8 @@ more important than the casting change itself.
 
 .. note::
 
-    As of the early NumPy 1.24 development branch, NumPy has preliminary and
-    limited support to test this proposal.  To try it, you can use the
-    development wheels from: https://anaconda.org/scipy-wheels-nightly/numpy
+    As of the NumPy 1.25.x series, NumPy has preliminary and limited support to
```

Comment:
```suggestion
    As of the NumPy 1.24.x series, NumPy has preliminary and limited support to
```
Actually, it was in the 1.24 dev branch, most of this works fine in 1.24 already (OTOH there are some additional fixes in 1.25).